scripts/rollup-options.js

```diff
@@ -59,10 +70,6 @@ const buildWithRollup = async ({
   rollUpCache = bundle.cache // store the cache object of the previous build
 
   await bundle.write({ format: 'es', file: outputFileJS })
-
```

Comment:
These snippets are intended to work with `babel-plugin-import` to let the styles to be loaded on demand. But it would be reasonable to be removed since I think we're not using it anymore